test/parsers/aircraftCommandParser/argumentParser.spec.js

```diff
@@ -157,3 +158,11 @@ ava('.timewarpParser() returns an array with 50 as a value when provided as an a
 
     t.true(result[0] === 50);
 });
+
+
+ava('.crossingParser() returns an array with the correct values', (t) => {
```

Comment:
This is the happy path test, which is good, but we should also have un-happy and/or WTF path tests in here too. 
 could you add at least one negative test case for this?

Reply:
No idea what to test for. Suggestions?

Reply:
We run a validator before the parser. Ideally, it catches all the invalid values. In case of #1249, it doesn't.